base/acme/Dockerfile

```diff
@@ -0,0 +1,93 @@
+#
+# Copyright Red Hat, Inc.
+#
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# https://docs.fedoraproject.org/en-US/containers/guidelines/guidelines/
+
+FROM registry.fedoraproject.org/fedora:30
+
+ARG NAME="pki-acme"
+ARG SUMMARY="Dogtag PKI ACME Responder"
+ARG LICENSE="GPLv2 and LGPLv2"
+ARG VERSION="0"
+ARG ARCH="x86_64"
+ARG MAINTAINER="Dogtag PKI Team <pki-devel@redhat.com>"
+ARG VENDOR="Dogtag"
+ARG COMPONENT="pki-core"
+ARG COPR_REPO=""
+
+LABEL name="$NAME" \
+      summary="$SUMMARY" \
+      license="$LICENSE" \
+      version="$VERSION" \
+      architecture="$ARCH" \
+      maintainer="$MAINTAINER" \
+      vendor="$VENDOR" \
+      usage="podman run -p 8080:8080 -p 8443:8443 $NAME" \
+      com.redhat.component="$COMPONENT"
+
+EXPOSE 8080 8443
+
+# Enable COPR repo if specified
+RUN if [ -n "$COPR_REPO" ]; then dnf install -y dnf-plugins-core; dnf copr enable -y $COPR_REPO; fi
+
+# Install packages
+RUN dnf install -y bind-utils iputils abrt-java-connector postgresql postgresql-jdbc dogtag-pki
```

Comment:
Actually, the ACME code is in `pki-server` package. We don't need `pki-ca` since this container will not be running the CA internally. However, I'm thinking to provide a front page so it doesn't look like a broken page. See https://acme.demo.dogtagpki.org. The front page should use the proper theme, that's why I included `dogtag-pki`, but maybe I can include `pki-server-theme` instead. What do you think?

Reply:
That page doesn't load for me (HTTP Status 404 – Not Found). I tried 8443 port too.

If adding theme is the only intention, I think `pki-server-theme` works. RHEL also has the same name, which makes the Dockerfile reusable.

Reply:
Right, that's what Endi wants to fix. I agree with fixing. Make 404 -> pretty page. Kinda like ACME: https://acme-v02.api.letsencrypt.org

Reply:
I've replaced `dogtag-pki` with `pki-server` for now. There are no theme packages at all on RHEL, so we might need to create a generic theme-less page. I'll do that in a separate PR.